python/ray/runtime_context.py

```diff
@@ -212,12 +212,98 @@ def get_task_id():
         return task_id.hex() if not task_id.is_nil() else None
 
     def _get_current_task_id(self) -> TaskID:
-        async_task_id = ray._raylet.async_task_id.get()
```

Comment:
would you mind explaining the change?

Reply:
chatted with @MengjinYan offline. The async case has already been handled by `get_current_task_id` in `_raylet.pyx`.